Tasks/UseDotNetV1/task.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
+    "name": "UseDotNet",
+    "friendlyName": "Use .NET Core",
+    "description": "Acquires a specific version of the .NET Core SDK from the internet or the local cache and adds it to the PATH. Additionally provides basic auth.",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "DotNetCore"
+    ],
+    "demands": [],
+    "ecosystem": ".NET Core",
+    "instanceNameFormat": "Use .NET Core",
+    "inputs": [
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "defaultValue": "1.0.4",
+            "required": true,
+            "helpMarkDown": "Version Spec of the version to use.  Examples: 10.x, 10.15.1, >=10.15.0"
+        },
+        {
+            "name": "auth",
+            "type": "string",
+            "label": "Auth",
+            "required": false,
+            "helpMarkDown": "The authentication feed/service point to use"
+        },
+        {
+            "name": "proxy",
+            "type": "boolean",
+            "label": "Proxy",
+            "defaultValue": true,
+            "required": false,
+            "helpMarkDown": "Whether or not to set up the .Net ecosystem to use the agent's proxy"
+        },
+        {
+            "name": "nuGetFeedType",
+            "type": "pickList",
+            "label": "NuGet Feed Type",
+            "helpMarkDown": "The type of NugetFeed used in the auth field. Can either be an internal feed or external service connection",
+            "required": false,
+            "options": {
+                "internal": "internal",
+                "external": "external"
```

Comment:
what's the default? external?

wouldnt this be interpreted from the auth info (e.g. properties in the typed-endpoint)

Reply:
This has changed a bit since you reviewed, I think this is good to have.